pkg/k8shandler/defaults.go

```diff
@@ -19,7 +19,4 @@ var (
 
 	defaultFluentdMemory     resource.Quantity = resource.MustParse("736Mi")
 	defaultFluentdCpuRequest resource.Quantity = resource.MustParse("100m")
-
```

Comment:
unrelated change?

Reply:
Nope, those are unused variables, they cause lint to fail. They are the reason I was pushed to do this - tired of getting other peoples lint in my PRs :)